assignment-2/src/model/question.js

```diff
@@ -0,0 +1,104 @@
+import { EventEmitter } from "events";
+import tag from "./tag";
+import author from "./user"
+
+class Question extends EventEmitter {
+    constructor() {
+        super();
+        this.state = {
+            questions: [{
+                id: 0,
+                title: "it's a title",
+                author: author.state.users[0],
+                text: "some text",
+                creationDate: "some date",
+                voteCount: 5,
+                tags: [tag.state.tags[1]]
+            }, {
+                id: 1,
+                title: "my question is",
+                author: author.state.users[0],
+                text: "this",
+                creationDate: "some date",
+                voteCount: 1,
+                tags: [ tag.state.tags[0], tag.state.tags[1] ]
+            }],
+            newQuestion: {
+                id: "",
+                title: "",
+                author: "",
+                text: "",
+                creationDate: "some date",
+                voteCount: "",
+                tags: ""
+            },
+            index: 2,
+            route: "questions_list"
+        };
+    }
+
+    findByTitle(title) {
+        let res = this.state.questions.filter(q => q.title.includes(title));
+        this.changeNewQuestionProperty("title", "");
```

Comment:
I guess this clears a input field on the UI right? This call should actually reside in the presenter --> you would not expect from the model that he clears a property when do call `findBy...` (which you would expect that is a read-only operation).